### PR TITLE
Translation fixes (description attached)

### PR DESCRIPTION
**Pull request by Laancelot**
- worked on a few translation fixes
- handled numbers in the proper French way
- handled some plurals
- fixed some conjugation
- fixed a few typos and misspelling
- harmonized some translations ("cliquez"  / "cliquer", etc.)
- added some translations where they didn't exist
Note:
- I only managed to go through a fifth of the doc (maybe)
- I'm a French native speaker
- I feel, apart from finishing to review the doc, that it would help to move around the translations in categories, like colors or country names

### CODE REVIEW

File: locales/fr_FR/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -20,32 +20,35 @@ msgstr ""
...
-msgstr "800,000 $ de donations privées !"
+msgstr "800 000 $ de dons privés !"
 
 msgctxt "SERP footer content"
 msgid "%s Billion Searches"
 msgstr "%s milliards de recherches"
 
 msgid "%s blocked by safe search."
-msgstr "%s bloqué par la recherche sécurisée."
+msgstr "%s bloqué par le module de recherche en toute sécurité."
```
Comment (GioSensation) [inline locales/fr_FR/LC_MESSAGES/duckduckgo.po:35]: The previous version seemed more accurate and was more succinct. Do you mind reviewing this again to make sure the meaning is correct? If I had to translate from French back to English, this new version would read something like "blocked by the search module, safely", which doesn't really match the original meaning.

File: locales/fr_FR/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -20,32 +20,35 @@ msgstr ""
 "X-Generator: Poedit 2.3\n"
 
 msgid "!Bang Search Shortcuts"
-msgstr "Raccourcis !Bang"
+msgstr "Raccourcis de recherche !Bang"
 
+#. input correct way to showcase amounts in French (done)
```
Comment (GioSensation) [inline locales/fr_FR/LC_MESSAGES/duckduckgo.po:25]: These comments don't belong in the file. If you want to specify why you made a change, you can add comments to the commit message or better still commit the file and add comments in the PR, like I am doing now. Please remove all these comments. Thanks for your help! 🙇 
```suggestion
```

File: locales/fr_FR/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -20,32 +20,35 @@ msgstr ""
...
 msgid "%s blocked by safe search."
-msgstr "%s bloqué par la recherche sécurisée."
+msgstr "%s bloqué par le module de recherche en toute sécurité."
 
 #. We realise this is the incorrect way to handle plurals. Sorry, we are working on a fix.
+#. fixed plurals
 msgctxt "published dates for organic results"
 msgid "%s days ago"
-msgstr "Il y a %s jours"
+msgstr "Il y a %s jour(s)"
```
Comment (GioSensation) [inline locales/fr_FR/LC_MESSAGES/duckduckgo.po:41]: This is not needed. This string will always be plural. The singular version can be found below and it's already translated. (Also remove the comment at line 38 as explained above) Thanks!
```suggestion
```

File: locales/fr_FR/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -20,32 +20,35 @@ msgstr ""
...
 #. in https://duckduckgo.com/params. multiple fors
 msgid "%s for %s"
 msgstr "%s pour %s"
 
 #. We realise this is the incorrect way to handle plurals. Sorry, we are working on a fix.
+#. fixed plurals
 msgctxt "published dates for organic results"
 msgid "%s hours ago"
-msgstr "Il y a %s heures"
+msgstr "Il y a %s heure(s)"
```
Comment (GioSensation) [inline locales/fr_FR/LC_MESSAGES/duckduckgo.po:51]: This is not needed. This string will always be plural. The singular version can be found below and it's already translated. (Also remove the comment at line 48 as explained above) Thanks!
```suggestion
```

**Review by GioSensation — CHANGES_REQUESTED** [review]
Thanks for your contribution, @Laancelot! I have left a few comments for you to address. Let me know if you have any question. I'll be happy to help.

File: locales/fr_FR/LC_MESSAGES/duckduckgo.po (hunk 3 above)
Comment (Laancelot) [reply locales/fr_FR/LC_MESSAGES/duckduckgo.po:39]: I'm solving this in the next commit to my current pull request. Thank you for the explanation

File: locales/fr_FR/LC_MESSAGES/duckduckgo.po (hunk 1 above)
Comment (Laancelot) [reply locales/fr_FR/LC_MESSAGES/duckduckgo.po:34]: I'm reverting that change as per your comment. Just to be noted French is a bit more "wordy" than English - less succinct

**Comment by Laancelot** [comment]
thanks, @GioSensation ! I have made the changes you requested
Out of curiosity, what do you think of my suggestion to rearrange the file by categorizing similar bits of text? (cities, colors, etc.)

**Review by GioSensation — APPROVED** [review]

**Comment by GioSensation** [comment]
@Laancelot Thanks for addressing this. Yes, we would like to sort files in a smarter way. Right now, they are sorted alphabetically by `msgid`, which doesn't help translators at all. It's on our radar. Thanks for the suggestion!

Merging now. 🎉

**Comment by Laancelot** [comment]
Alright, thank you @GioSensation ! I am planning to keep on reviewing this file if you're OK with this!

**Pull request merged**